app/javascript/components/patient/Patient.js

```diff
@@ -250,152 +401,268 @@ class Patient extends React.Component {
                 </Row>
                 <Row>
                   <Col className="text-truncate">
-                    <b>Type:</b>{' '}
-                    <span>{this.props.details.additional_planned_travel_type ? `${this.props.details.additional_planned_travel_type}` : '--'}</span>
-                    <br />
-                    <b>Place:</b>{' '}
-                    <span>
-                      {`${
-                        this.props.details.additional_planned_travel_destination_country ? this.props.details.additional_planned_travel_destination_country : ''
-                      }`}
-                      {`${
-                        this.props.details.additional_planned_travel_destination_state ? this.props.details.additional_planned_travel_destination_state : ''
-                      }`}
-                      {!this.props.details.additional_planned_travel_destination_country &&
-                        !this.props.details.additional_planned_travel_destination_state &&
-                        '--'}
-                    </span>
-                    <br />
-                    <b>Port Of Departure:</b>{' '}
-                    <span>{`${
-                      this.props.details.additional_planned_travel_port_of_departure ? this.props.details.additional_planned_travel_port_of_departure : '--'
-                    }`}</span>
-                    <br />
-                    <b>End Date:</b>{' '}
-                    <span>{`${
-                      this.props.details.additional_planned_travel_start_date
-                        ? moment(this.props.details.additional_planned_travel_start_date, 'YYYY-MM-DD').format('MM/DD/YYYY')
-                        : '--'
-                    }`}</span>
-                    <br />
-                    <b>Start Date:</b>{' '}
-                    <span>{`${
-                      this.props.details.additional_planned_travel_end_date
-                        ? moment(this.props.details.additional_planned_travel_end_date, 'YYYY-MM-DD').format('MM/DD/YYYY')
-                        : '--'
-                    }`}</span>
+                    {!this.props.details.additional_planned_travel_type && <span className="none-text">None</span>}
+                    {this.props.details.additional_planned_travel_type && (
+                      <React.Fragment>
+                        <Row>
+                          <Col>
+                            <b>Type:</b>{' '}
+                            <span>{this.props.details.additional_planned_travel_type ? `${this.props.details.additional_planned_travel_type}` : '--'}</span>
+                          </Col>
+                        </Row>
+                        <Row>
+                          <Col>
+                            <b>Place:</b>{' '}
+                            <span>
+                              {`${
+                                this.props.details.additional_planned_travel_destination_country
```

Comment:
Couldn't this be `{this.props.details.additional_planned_travel_destination_country}` ? Cause if it's falsey, it'll be `''` implicity, no?

Reply:
you're correct on this - fixed